contracts/PreConfirmations.sol

```diff
@@ -391,7 +420,7 @@ contract PreConfCommitmentStore is Ownable {
      * @dev Initiate a slash for a commitment.
      * @param commitmentIndex The hash of the commitment to be slashed.
      */
-    function initiateSlash(bytes32 commitmentIndex) public onlyOracle {
+    function initiateSlash(bytes32 commitmentIndex, uint256 residualDecayAfterBid) public onlyOracle {
```

Comment:
I assume you mean `residualBidAfterDecay` here? Also same comment as above usage of this term